src/utils/calibration.test.ts

```
@@ -0,0 +1,139 @@
+describe('calibration utils', () => {
+  beforeEach(() => {
+    Object.defineProperty(globalThis, 'localStorage', {
+      value: createLocalStorageMock(),
+      configurable: true,
+      writable: true,
+    });
+  });
+
+  beforeEach(() => {
+    localStorage.clear();
+  });
```

Biome (2.4.4)

[error] Duplicate beforeEach hook found.

(lint/suspicious/noDuplicateTestHooks)